Kickstarter-iOS/Views/Cells/SettingsNotificationCell.swift

```diff
@@ -90,13 +94,12 @@ final class SettingsNotificationCell: UITableViewCell, NibLoading, ValueCell {
   override func bindViewModel() {
     super.bindViewModel()
 
+    self.rac.accessibilityValue = viewModel.outputs.projectCountText
     self.emailNotificationsButton.rac.selected = viewModel.outputs.emailNotificationsEnabled
     self.emailNotificationsButton.rac.hidden = viewModel.outputs.emailNotificationButtonIsHidden
     self.projectCountLabel.rac.text = viewModel.outputs.projectCountText
     self.pushNotificationsButton.rac.selected = viewModel.outputs.pushNotificationsEnabled
     self.pushNotificationsButton.rac.hidden = viewModel.outputs.pushNotificationButtonIsHidden
-    self.projectCountLabel.rac.accessibilityHint = viewModel.outputs
```

Comment:
I wasn't able to make VoiceOver read the hint (maybe this needs more testing on an actual device with different OS version). @Scollaco could you please run AppStore version on iOS 10/11 and see if this hint is being spoken?

If not we should also be able to remove `profile_project_count_projects_backed` from the `Strings` file

Reply:
I was able to make the VoiceOver read the hint once I changed `self.projectCountLabel.rac.accessibilityHint` to `self.rac.accessibilityHint`, because than the hint would be coupled to the cell and not to the label.

Reply:
Interesting.

Not really sure why it won't use hint on a `UILabel`. But anyways in order to follow the native Settings.app patterns and the a11y audit, I think we should remove this.

@ifbarrera since you've added this localized hint, do you feel strongly about it?

Reply:
@dusi nope don't feel strongly about it, you can go ahead and remove it for consistency with the rest of Settings accessibility work!